worker/lease/manager.go

```diff
@@ -21,10 +21,21 @@ var logger = loggo.GetLogger("juju.worker.lease")
 // the manager has started (and possibly finished) shutdown.
 var errStopped = errors.New("lease manager stopped")
 
+type dummySecretary struct{}
```

Comment:
We create a "dummy" `Secretary` to supplement the `DeadManager`. Some methods on `DeadManager` where panicking due to a nil Secretary. My initial reaction was to add an interface over Manager where all methods return the error as specified in `NewDeadManager`s comment, however I cannot be sure if an interface was not used initially because of a need for side effects when methods are called.